Yosemite/Yosemite.xcodeproj/project.pbxproj

```diff
@@ -1109,27 +1109,27 @@
 			path = Base;
 			sourceTree = "<group>";
 		};
-		B5C9DE1D2087FF20006B910A /* Mockups */ = {
+		B5C9DE1D2087FF20006B910A /* Mocks */ = {
 			isa = PBXGroup;
 			children = (
 				578CE7892475E78C00492EBF /* Networking */,
 				D8BD6A4B229D07C8007CAD6C /* custom-shipment-provider.plist */,
 				D87F615F2265B2400031A13B /* shipment-provider.plist */,
-				B5C9DE1E2087FF20006B910A /* MockupProcessor.swift */,
-				B5C9DE202087FF20006B910A /* MockupAcount.swift */,
-				02FF055523D984310058E6E7 /* MockupFileManager.swift */,
+				B5C9DE1E2087FF20006B910A /* MockProcessor.swift */,
```

Comment:
💡  Perhaps naming it as `MockActionsProcessor` would be better since it conforms to `ActionsProcessor`. 🙂

Reply:
I like this suggestion, good catch! Addressed it in https://github.com/woocommerce/woocommerce-ios/pull/3280/commits/db6db4c778f189c1e63454ed4817644541a04a87

Reply:
Awesome! Merge away! 🚢